frontend/src/components/dashboard/Dashboard.js

```diff
@@ -14,13 +14,16 @@ import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
 import AssignmentIcon from "@material-ui/icons/Assignment";
 import EmojiObjectsIcon from "@material-ui/icons/EmojiObjects";
 import GroupAddIcon from "@material-ui/icons/GroupAdd";
-import React, { useContext, useRef, useState } from "react";
+import React, { useContext, useEffect, useRef, useState } from "react";
 import { getLocalePrefix } from "../../../public/lib/apiOperations";
 import getTexts from "../../../public/texts/texts";
 import theme from "../../themes/theme";
 import OpenClimateMatchButton from "../climateMatch/OpenClimateMatchButton";
 import UserContext from "../context/UserContext";
 import UserImage from "./UserImage";
+import CreateIdeaDialog from "../ideas/createIdea/CreateIdeaDialog";
+import { getUserOrganizations } from "../../../public/lib/organizationOperations";
+import Cookies from "universal-cookie";
```

Comment:
Nit: do you mind colocating this 3rd party import to the others, up with line 18? And then delineating relative imports and 3rd party with a newline? I find this helps aid with readability some :-)

Reply:
Sure, no problem. :-)